tox.ini

```diff
@@ -0,0 +1,21 @@
+# content of: tox.ini, put in same dir as setup.py
+[tox]
+envlist=pep8,py34
+
+[testenv]
+setenv=
+    SIDEBOARD_CONFIG_OVERRIDES=development-defaults.ini;test-defaults.ini
+deps= -rrequirements_test.txt
+whitelist_externals=
+    /bin/ln
+    /bin/rm
+commands=
+    rm -f {envdir}/src/sideboard/plugins/uber
+    ln -s {toxinidir} {envdir}/src/sideboard/plugins/uber
```

Comment:
Here we are making a symbolic link from the sideboard plugins directory back to the ubersystem project directory. We are, in effect, installing the uber plugin in sideboard.